app/models/concerns/foreman_templates/provisioning_template_import.rb

```diff
@@ -3,9 +3,9 @@ module ProvisioningTemplateImport
     extend ActiveSupport::Concern
 
     module ClassMethods
-      def import!(name, text, metadata)
+      def import!(name, text, metadata, force)
```

Comment:
I'm afraid that changing this API makes importing of JobTemplates incompatbile again. If that's the case, could you please also prepare a PR for rex (and openscap seed)? We could maybe make force argument optional, defaulting to false which would avoid the need for openscap seed change.